src/group_theory/perm/basic.lean

```diff
@@ -176,6 +176,9 @@ begin
   simpa using equiv.congr_fun h i
 end
 
+@[simp] lemma perm_congr_eq_mul (e p : perm α) :
+  e.perm_congr p = e * p * e⁻¹ := rfl
```

Comment:
I had to think a bit to realize this whould indeed be a good `simp` lemma. Let's explain why in a comment:
```suggestion
-- If `e` is also a permutation, we can write `perm_congr` completely
-- in terms of the group structure.
@[simp] lemma perm_congr_eq_mul (e p : perm α) :
  e.perm_congr p = e * p * e⁻¹ := rfl
```